src/App.tsx

```diff
@@ -1,24 +1,371 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import cn from 'classnames';
+import {
+  SetStateAction,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './Components/Header';
+import { Section } from './Components/Section';
+import { Footer } from './Components/Footer';
+import { Todo } from './types/Todo';
+import * as todoService from './api/todos';
+import { filterTodos } from './utils/services';
 
-const USER_ID = 0;
+const USER_ID = 12083;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [erroMessage, setErrorMessage] = useState('');
+  const [statusTodo, setStatusTodo] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [loadingTodos, setLoadingTodos] = useState<Todo[] | null>(null);
```

Comment:
why do you need to save array of whole todo object, you can just save the todo.id in this array